rayon-core/README.md

```diff
@@ -8,4 +8,4 @@ Please see [Rayon Docs] for details about using Rayon.
 
 [Rayon Docs]: https://docs.rs/rayon/
 
-Rayon-core currently requires `rustc 1.36.0` or greater.
+Rayon-core currently requires `rustc 1.46.0` or greater.
```

Comment:
Technically, `rayon-core` is unaffected here, though it will be untested unless we add another CI job.

But I intend to bump to 2021 in about a month (#964), so I probably won't publish until then.